src/colmap/exe/sfm.cc

```diff
@@ -456,6 +456,7 @@ int RunPointTriangulatorImpl(
   //////////////////////////////////////////////////////////////////////////////
 
   const auto tri_options = mapper_options.Triangulation();
+  const auto map_options = mapper_options.Mapper();
```

Comment:
Should we update the naming here? Now that in this exe/sfm.cc ``mapper_options`` refers to a instance that is in type ``IncrementalMapperOptions``, while inside the controller ``mapper_options`` refers to ``map_options`` here (in type ``IncrementalMapper::Options``). Maybe we should update all the ``mapper_options`` inside the controller to ``map_options`` for consistency, to avoid potential confusion :)

Reply:
I don't think that is matters much. We could instead rename `mapper_options` to `controller_options` in this function only. I anyway plan to move this function elsewhere, for example into the `IncrementalMapperController`.

Reply:
``controller_options`` sounds good!